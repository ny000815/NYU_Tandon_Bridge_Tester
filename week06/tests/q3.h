#ifndef GENERIC_HW6_Q3_H
#define GENERIC_HW6_Q3_H

double eApprox(int n);

#endif
