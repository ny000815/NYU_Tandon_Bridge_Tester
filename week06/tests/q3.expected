n = -1	0.000000000000000000000000000000
n = 0	1.000000000000000000000000000000
n = 1	2.000000000000000000000000000000
n = 2	2.500000000000000000000000000000
n = 5	2.716666666666666341001246109954
n = 10	2.718281801146384513145903838449
n = 20	2.718281828459045534884808148490
n = 50	2.718281828459045534884808148490
n = 100	2.718281828459045534884808148490
n = 170	2.718281828459045534884808148490
n = 171	2.718281828459045534884808148490
